src/dataframe/dataframe.jl

```diff
@@ -98,10 +98,32 @@ type DataFrame <: AbstractDataFrame
     end
 end
 
+# Return and AbstractVector of length `len` filled with `x`.
+# `x` is recycled if `len(x)` is an even multiple of `len`.
+rep_len(x, len) = rep(x, len)
+function rep_len(x::AbstractVector, len)
```

Comment:
I'd rather raise an error than return a vector of a different length than the requested one:
```julia
julia> rep_len([1, 2], 7)
6-element Array{Int64,1}:
 1
 2
 1
 2
 1
 2
```